### ISSUE TITLE
error: unused function 'font_path' [-Werror,-Wunused-function]

### ISSUE THREAD

**Issue by rbruders** [libgd/libgd#314]
when I make my file I keep running into this issue where I make it until I get an error on this line 

```
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I/usr/local/Cellar/libpng/1.6.25/include/libpng16 \
  -Werror -g -O2 -fvisibility=hidden -Wall -MT gdft.lo -MD -MP -MF .deps/gdft.Tpo -c gdft.c  \
  -fno-common -DPIC -o .libs/gdft.o
gdft.c:1634:15: error: unused function 'font_path' [-Werror,-Wunused-function]
```

**Comment by vapier** [comment]
assuming you're using a release (since you didn't say what version you're building or where or your configure or config.log settings), this has been fixed in git already

**Comment by cmb69** [comment]
You can use `./configure --disable-werror` to work around the issue, until GD 2.2.4 will be released.

**Comment by vapier** [comment]
hmm, it wasn't my intention to `--enable-werror` by default in releases.  i'll fix that.

**Comment by vapier** [comment]
i fixed it via #339
